### ISSUE TITLE
Crear página para eventos

### ISSUE THREAD

**Issue by satanas** [pyve/pyve.github.com#74]
Debería recopilar el historial de todos los eventos realizados y apuntar a http://pyve.github.io/eventos

**Comment by atmantree** [comment]
tengo la tentación de hacer 2 repos adicionales, uno para eventos y otro para la fundación? con ghpages tendríamos igualmente la url, pero un repo específico para los documentos relacionados, te convence organizarnos así?

**Comment by nhomar** [comment]
El 4 de febrero de 2016, 11:41, Carlos Gustavo Ruiz <
notifications@github.com> escribió:

> tengo la tentación de hacer 2 repos adicionales, uno para eventos y otro
> para la fundación? con ghpages tendríamos igualmente la url, pero un repo
> específico para los documentos relacionados, te convence organizarnos así?
>

a mi me gusta todo en uno.

Hacer split de repos complicará la cosa, y su administración.




-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.


[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>

**Comment by satanas** [comment]
De acuerdo con @nhomar, prefiero todo en uno para facilitar el mantenimiento

**Comment by atmantree** [comment]
perfecto entonces..

**Comment by atmantree** [comment]
para tener en cuenta, crear una página para los eventos PyConVE y 1er Encuentro (actualmente están en la página de Comunidad).

**Comment by jackboot7** [comment]
Se puede tener todo en un sólo repo organizado en directorios.

**Comment by nhomar** [comment]
2016-02-04 13:46 GMT-06:00 Luis Alberto Santana <notifications@github.com>:

> Se puede tener todo en un sólo repo organizado en directorios.
>

Uno por directorio, para eso son los trees... así se mantiene una imagen
estándar.

Lo que se puedes es creer un directorio eventos/

Y dentro un html por evento.


-- 
--------------------
Saludos Cordiales

CEO at Vauxoo <https://www.vauxoo.com> Odoo's Gold Partner.


[image: --]
Nhomar Hernandez
[image: http://]about.me/nhomar
<http://about.me/nhomar?promo=email_sig>

**Comment by jgomo3** [comment]
Pienso que no está demás que cada evento mantenga su propia página y mantenida por sus organizadores.

Entonces, en http://python.org.ve/, en la sección eventos, pudiéramos tratar de mantener lo mejor posible un índice de todos los eventos del país.

Fíjense en http://python.org/, la sección de eventos es como lo describí. De hecho, tienen una sección para [eventos python](https://www.python.org/events/python-events/) (como pycon) y otro para [eventos de grupos de usuarios](https://www.python.org/events/python-user-group/) (como grupos de Django).

Pero Pycon tiene su propia página dedicada sólo a ese esfuerzo.

**Nota:** Algo curioso que noté, es que la págia de [PyCon](http://www.pycon.org/) tiene entre su lista de regiones a Venezuela con este URL: http://ve.pycon.org/ (que no sirve). Y si se fijan, la gran mayoría de los PyCon tienen ese dominio **"pycon.org"**.

**Comment by mamcode** [comment]
Estoy de acuerdo con @jgomo3 tener una página de eventos con el listado de eventos realizados, si se hacen eventos más específicos por ejemplo eventos sobre Django, odoo, Pyramid, etc.. pues veo también valido tener otra sección para eventos de grupos de usuarios, aunque no si se hayan muchos eventos en la actualidad de grupos usuarios...

Respecto a mantener una página por evento lo veo válido, pues cada uno tendrá sus particularidades, de hecho en python.org lo hacen, fijense el [PyCaribbean](http://pycaribbean.com/), tiene su propia página mantenida por los organizadores supongo! Si lo que se quiere es mantener el mismo estilo (tema CSS) para todos los eventos se podría diseñar una plantilla que sirva de base, pero yo particularmente no lo considero necesario...

Mis dos bolos :-)

**Comment by hernanramirez** [comment]
en el index.html hay un link que apunta a /eventos/ pero genera error, me imagino por no tener index.html ¿se crea uno que liste los eventos?
